FormalConjectures/Wikipedia/Mersenne.lean

```diff
@@ -85,4 +85,12 @@ theorem new_mersenne_conjecture.variants.prime (p : ℕ) (hp : p.Prime) :
     NewMersenneConjectureStatement p := by
   sorry
 
-end NewMersenne
+/--
+Are there infinitely many Mersenne primes?
+-/
+@[category research open, AMS 11]
+theorem infinitely_many_mersenne_primes :
+  Set.Infinite { p : ℕ | p.GivesMersennePrime } ↔ answer(sorry) := by
```

Comment:
I personally also like `Set.Infinite (GivesMersennePrime : Set ℕ)`, but I think there is not consensus for this here.